Devildog4355/devildog_BWL.sql

```diff
@@ -0,0 +1,215 @@
+UPDATE creature_template SET ScriptName='boss_halion_twilight' WHERE entry=40142;
+
+-- 2801
+UPDATE creature_template SET ScriptName='npc_dark_matter' WHERE entry=28235;
+UPDATE creature_template SET ScriptName='npc_searing_gaze' WHERE entry=28265;
+
+-- 2802
+UPDATE creature_template SET ScriptName='' WHERE entry=29682;
+
+-- 2806
+UPDATE creature_template SET ScriptName='spell_dummy_npc' WHERE entry=21729;
+
+-- 2808
+UPDATE creature_template SET ScriptName='boss_varos' WHERE entry=27447;
+
+-- 2809
+UPDATE creature_template SET ScriptName='npc_azure_ring_captain' WHERE entry=28236;
+UPDATE creature_template SET ScriptName='npc_arcane_beam' WHERE entry=28239;
+UPDATE creature_template SET ScriptName='npc_centrifuge_core' WHERE entry=28183;
+DELETE FROM scripted_event_id WHERE id IN (10665,12229,18454,18455);
+INSERT INTO scripted_event_id VALUES
+(10665,'event_spell_call_captain'),
+(12229,'event_spell_call_captain'),
+(18454,'event_spell_call_captain'),
+(18455,'event_spell_call_captain');
+
+-- 2810
+UPDATE creature_template SET ScriptName='npc_planar_anomaly' WHERE entry=30879;
+
+-- 2815
+UPDATE creature_template SET ScriptName='npc_oculus_drake' WHERE entry IN (27756, 27692, 27755);
+
+-- 2816
+UPDATE creature_template SET ScriptName='npc_crystal_spike_trigger' WHERE entry IN (27101, 27079);
+
+-- 2817
+UPDATE creature_template SET ScriptName='npc_grauf' WHERE entry=26893;
+
+-- 2821
+UPDATE creature_template SET ScriptName='spell_dummy_npc' WHERE entry=13016;
+
+-- 2823
+UPDATE creature_template SET ScriptName='npc_impale_target' WHERE entry=29184;
+
+-- 2826
+UPDATE creature_template SET ScriptName='npc_wyrmrest_skytalon' WHERE entry=30161;
+
+-- 2831
+UPDATE gameobject_template SET ScriptName='go_father_flame' WHERE entry=175245;
+
+-- 2832
+UPDATE creature_template SET ScriptName='boss_krick' WHERE entry=36477;
+UPDATE creature_template SET ScriptName='boss_ick' WHERE entry=36476;
+UPDATE creature_template SET ScriptName='npc_exploding_orb' WHERE entry=36610;
+
+-- 2837
+UPDATE creature_template SET ScriptName='npc_ymirjar_deathbringer' WHERE entry=36892;
+DELETE FROM scripted_areatrigger WHERE entry=5578;
+INSERT INTO scripted_areatrigger VALUES (5578,'at_pit_of_saron');
+
+-- 2838
+UPDATE creature_template SET ScriptName='npc_collapsing_icicle' WHERE entry=36847;
+
+-- 2840
+UPDATE creature_template SET ScriptName='boss_tyrannus' WHERE entry=36658;
+UPDATE creature_template SET ScriptName='boss_rimefang_pos' WHERE entry=36661;
+
+-- 2842
+DELETE FROM scripted_areatrigger WHERE entry IN (5581);
+INSERT INTO scripted_areatrigger VALUES
+(5581,'at_pit_of_saron');
+
+-- 2846
+UPDATE creature_template SET ScriptName='' WHERE entry=15302;
+UPDATE creature_template SET ScriptName='' WHERE entry=15260;
+
+-- 2847
+UPDATE creature_template SET ScriptName='npc_erekem_guard' WHERE entry=32228;
+
+-- 2856
+UPDATE gameobject_template SET ScriptName='go_black_dragon_egg' WHERE entry=177807;
+
+-- Cleanup
```

Comment:
also remove the cleanup, i sent it to X-Savior directly since he can only approve this ^^